Prevents "g++: error: unrecognized command-line option ‘--target=aarch64-linux-android24’"

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,7 +20,6 @@ else ()
 endif ()
 
 # Compiler settings (this must come before calling project)
-set(CMAKE_CXX_COMPILER g++)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED on)
 
